components/migrate/MigrateIndex/MigrateContentWaiting.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="mt-8 pt-4">
+    <div class="is-flex is-relative section-title">
+      <img src="/migrate/state-waiting.svg" alt="Ready" />
+      <p>{{ $t('migrate.waiting.title') }}</p>
+    </div>
+
+    <div class="has-text-grey mt-2">
+      {{ $t('migrate.waiting.desc') }}
+    </div>
+
+    <div class="collection">
+      <div v-for="cl in collections" :key="cl.id" class="collection-card">
+        <div
+          class="collection-card-banner"
+          :style="{
+            backgroundImage: `url(${sanitizeIpfsUrl(cl.meta?.image)})`,
+          }"></div>
+        <div
+          class="collection-card-avatar"
+          :style="{
+            backgroundImage: `url(${sanitizeIpfsUrl(cl.meta?.image)})`,
+          }"></div>
+
+        <div class="collection-card-info">
+          <p class="is-size-5 has-text-weight-bold">{{ cl.name }}</p>
+          <p>
+            <span class="has-text-grey mr-2">
+              {{ $t('migrate.waiting.status') }}
+            </span>
+            <a href="#!" class="has-text-k-blue">Another nice name </a>
```

Comment:
nice name

Reply:
haha. will open a followup PR regarding this. ref: https://github.com/kodadot/nft-gallery/issues/7560#issuecomment-1779513052